iOS/README.md

```diff
@@ -0,0 +1,145 @@
+<p align="center">
+  <img src="../assets/UNiDAYS_Logo.png" />
+</p>
+<br/>
+
+# UNiDAYS Swift Coding Style Guide
+[Code Formatting](#code-formatting)
+  * [Spacing](#spacing)
+  * [Line Length](#line-length)
+  * [Braces](#braces)
+  * [Semicolons](#semicolons)
+
+[Code Styles](#code-styles)
+  * [Enums](#enums)
+  * [Type Inference](#type-inference)
+  * [Syntactic Sugar](#type-inference)
+  * [Happy Path](#happy-path)
+
+## Code Formatting
+### Spacing
+Use 4 spaces for tabs. Ensure that there is a no trailing whitespace anywhere and ensure that there is a newline at the end of every file. All of these can be configured in `Xcode -> Preferences -> Text Editing`.
+
+### Line Length
+Each line should have a maximum of 160 characters per line, you may find it useful to setup page guides `Xcode -> Preferences -> Text Editing -> Page Guide at column`.
+
+### Braces
+Open braces on the same line as the statement but close on a new line.
+#### Preferred:
+```
+if user.isStudent {
+    // Do something
+}
+```
+
+#### Not Preferred:
+```
+if user.isStudent
+{
+    // Do something
+}
+```
+
+### Semicolons
+Semicolons are only required if you wish to combine multiple statements on a single line. Do not write multiple statements on a single line separated with semicolons.
+
+#### Preferred:
+```
+let x = 1
+let y = 2
+```
+
+#### Not Preferred:
+```
+let x = 1; let y = 2;
+```
+
+## Code Styles
+### Enums
+Enums should use upper-case first letter camel casing for the type name and each case should have its own line and should be camcel cased. Where practical, enums should have their values explicitly declared.
+#### Preferred:
+```
+enum MyEnum {
+    case aRedEnum
+    case aBlueEnum
+}
+
+enum SomeEnum: Int {
+    case aRedEnum = 1
+    case aBlueEnum = 2
+}
+```
+
+#### Not Preferred:
+```
+enum myEnum {
+    case aRedEnum
+    case aBlueEnum
+}
+
+enum SomeEnum: Int {
+    case ARedEnum = 1
+    case ABlueEnum = 2
+}
+```
+
+### Type Inference
+Let the compiler infer the type for constants. Only specify the specific type when required, such as `CGFloat`.
+#### Preferred:
+```
+let name: String = "UNiDAYS"
```

Comment:
This line doesn't seem to be following the rules immediately above it.

Reply:
Oops. Made requested changes.